controller/named_work_items.go

```diff
@@ -0,0 +1,38 @@
+package controller
+
+import (
+	"fmt"
+
+	"github.com/fabric8-services/fabric8-wit/app"
+	"github.com/fabric8-services/fabric8-wit/application"
+	"github.com/fabric8-services/fabric8-wit/jsonapi"
+	"github.com/fabric8-services/fabric8-wit/rest"
+	"github.com/goadesign/goa"
+	errs "github.com/pkg/errors"
+)
+
+// NamedWorkItemsController implements the named_work_items resource.
+type NamedWorkItemsController struct {
+	*goa.Controller
+	db application.DB
+}
+
+// NewNamedWorkItemsController creates a named_work_items controller.
+func NewNamedWorkItemsController(service *goa.Service, db application.DB) *NamedWorkItemsController {
+	return &NamedWorkItemsController{
+		Controller: service.NewController("NamedWorkItemsController"),
+		db:         db,
+	}
+}
+
+// Show shows a work item form the given named space and its number
```

Comment:
well, I thought that `named space` would already assume `space name + owner's username`. I added it in the comment for more expressiveness